extensions/sockets/src/main/resources/META-INF/mule-module.properties

```diff
@@ -1,6 +1,23 @@
-module.name=sockets
+module.name=sockets-extension
 
 artifact.export.classPackages=org.mule.module.socket.api,\
+                              org.mule.module.socket.api.client,\
+                              org.mule.module.socket.api.config,\
+                              org.mule.module.socket.api.connection,\
+                              org.mule.module.socket.api.connection.tcp,\
+                              org.mule.module.socket.api.connection.udp,\
+                              org.mule.module.socket.api.exceptions,\
+                              org.mule.module.socket.api.protocol,\
+                              org.mule.module.socket.api.provider,\
+                              org.mule.module.socket.api.socket,\
+                              org.mule.module.socket.api.socket.tcp,\
+                              org.mule.module.socket.api.socket.udp,\
+                              org.mule.module.socket.api.source,\
                               org.mule.module.socket.internal,\
+                              org.mule.module.socket.internal.metadata,\
```

Comment:
I thought we needed to expose all the packages until a certain (don't remember exactly which one) class loader issue was solved.